cmd/cluster-provisioner/main.go

```diff
@@ -29,71 +30,71 @@ func main() {
 
 	provisioner, err := pv.GetCloudProvisioner(cloudProvider, provisionPropsFile)
 	if err != nil {
-		fmt.Println(err)
+		log.Error(err)
 		os.Exit(1)
```

Comment:
FYI. Logrus looks like providing `log.Fatal` for this code pattern.
https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

Reply:
@yoheiueda yes, Fatal is right approach here, I'll fix it. Thank you!